docs/src/js/components/ModulesDrawer.react.js

```diff
@@ -975,6 +974,7 @@ export default class ModulesDrawer extends React.Component {
     }
 
     _onPathDrawerClick(path) {
+        console.log('path', path);
```

Comment:
If this is a useful `console.log()` message on the documentation page for demonstrating functionality, should we put the the ESLINT suppression comment?  If this was for temporary debugging only, let's please remove it altogether.

Reply:
It was/is to show something happened to the dev.

Reply:
Yep, so that is a useful log message, on a documentation page.  So let's keep it and exempt it from ESLINT, right?

Reply:
Nope, I was wrong, totally not needed.